### PR TITLE
Rename geo-countries to geo-maps

### PR DESCRIPTION
**Pull request by simonepri**
I've worked on the project recently and I've decided to rebrand it with a more concise name.
I've also added [26 new maps](https://github.com/simonepri/geo-maps)! 🎉

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ GeoJSON utilities that will make your life easier.
 * [natural earth](http://www.naturalearthdata.com/): country, province, and geographical data
 * [geojson.xyz](http://geojson.xyz/): natural earth data in web-friendly sizes with hotlinking
 * [world-atlas](https://github.com/topojson/world-atlas): customizable simplified versions of natural earth data
-* [geo-countries](https://github.com/simonepri/geo-countries): GeoJSON political boundaries from OSM
+* [geo-maps](https://github.com/simonepri/geo-maps): High Quality GeoJSON maps programmatically generated.
```
Comment (tmcw) [inline README.md]: This is kind of a less good description than the previous one - in a list of maps of countries, or cities, or airplane routes, this doesn't help people know what the maps are of. Does 'political boundaries' no longer describe the scope of the project?

Comment (simonepri) [reply README.md]: You are perfectly right! The idea is to expand the scope of the project to more than only 'political boundaries' maps.

Now it contains maps like [this one](http://mapshaper.org/?files=https://unpkg.com/@geo-maps/world-land-100m/map.geo.json) that are not strictly related to political boundaries.
In fact I hope to be able to populate it with any other kind of useful map that could be generated using open datasets.

Have you any suggestions for a proper description?

Comment (tmcw) [reply README.md]: I wish I did, but nothing better comes to mind. I'll merge now and let me know if the tagline changes in the future. 😝

Comment (simonepri) [reply README.md]: Thank you @tmcw!

**Pull request merged**